src/Sql/PostgreSQL/Functions/user_delete_by_id.sql

```diff
@@ -0,0 +1,71 @@
+DROP FUNCTION IF EXISTS user_delete_by_id (UUID);
+
+CREATE OR REPLACE FUNCTION user_delete_by_id 
+(
+    _id UUID
+)
+RETURNS VOID
+LANGUAGE 'plpgsql'
+AS
+$BODY$
+BEGIN
+    -- functions dont support commit/rollback transactions, only v11+ procedures can do this
```

Comment:
well that sucks

Reply:
@Papina why not use a procedure then?

Reply:
@mnuccioarpae 
I don't know Dapper, but apparently it cant use procedures

Reply:
@Papina Me neither, but it looks strange to me since CALL conforms to the SQL standard.